src/System.Private.ServiceModel/src/System/ServiceModel/Channels/HttpChannelFactory.cs

```diff
@@ -647,42 +648,182 @@ public void Fault(RequestChannel channel)
                             throw;
                         }
                     }
+                }
 
-                    try
+                private bool PrepareMessageHeaders(Message message)
```

Comment:
It's still difficult to know why we're seeing both red and green for code that came from somewhere else.  I'm not sure what I can ignore and what I need to review carefully.  Anything you can do to identify areas we can ignore would be a huge help.